app/views/scan_analytics/_index.html.haml

```diff
@@ -0,0 +1,69 @@
+:ruby
+  scan_data = @scan_data.present? ? @scan_data : scan_data
+  scan_data = JSON.parse(scan_data.data.to_json, object_class: OpenStruct)
+  @cwes = scan_data.cwe
```

Comment:
Instead of instance variable, use a local variable for `cwes` and pass it to the partial.